nhlpy/api/schedule.py

```diff
@@ -29,13 +29,13 @@ def get_schedule_by_team_by_month(self, team_abbr: str, month: Optional[str] = N
         resource = f"club-schedule/{team_abbr}/month/{month if month else 'now'}"
         return self.client.get(resource=resource).json()["games"]
 
-    def get_schedule_by_team_by_week(self, team_abbr: str) -> List[dict]:
+    def get_schedule_by_team_by_week(self, team_abbr: str, date: Optional[str] = None) -> List[dict]:
         """
-        This returns the schedule for the team (team_abbr) for the current week.
+        This returns the schedule for the team (team_abbr) for the week set, or the current week if no week is specificed. should be passed "2024-02-10" as an ex.

```

Comment:
```suggestion
        This returns the schedule for the team (team_abbr) for the week set, or the current week if no week is specified. 
        
        :param date: Optional date, in which to search the week of.  In format YYYY-MM-DD.  Example "2024-02-10".

```

Reply:
This looks good.  Just need to update the doc string.  Also, dont forget to run `ruff .` locally, to run a lint check.  That is part of the build pipeline.

```
$ poetry run ruff .
nhlpy/api/schedule.py:34:122: E501 Line too long (162 > 121 characters)
```